readme.txt
=== iframe popup ===
Contributors: gopiplus, www.gopiplus.com
Donate link: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Author URI: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Plugin URI: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Tags: iframe, popup
Requires at least: 3.4
Tested up to: 4.3
Stable tag: 1.8
License: GPLv2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html

Iframe popup plugin is specially developed to display any webpage in the popup window using web url. It uses fancybox to display popup in iframe.

== Description ==

Check official website for live demo [http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/)

*   [Live Demo](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/ "Live Demo")
*   [More Informations](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/ "More Informations")
*   [Suggestion](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/ "Suggestion")
*   [Video Tutorial](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/ "Video Tutorial")

Iframe popup plugin is specially developed to display any webpage in the popup window using web URL. Iframe popup uses JQuery fancybox extension to display popup in iframe window. This plugin will help you to display popup window easily in your blog. You can easily customize the fancybox popup attributes in the plugin admin page.

= Plugin Features: =

*   Lightweight JQuery (fancybox extension).
*   Timeout to open popup.
*   Option to configure once per session.
*   Responsive.

= Frequently Asked Questions: =

*   How to show popup in specific page/post?
*   How to show popup in entire website?
*   How to show popup only on home page?
*   How to show popup in entire website except home page?
*   How to display iframe popup window once per session?
*   What is CATEGORY and ID in the plugin short code?
*   How to set/modify popup window size?
*   How to hide/disable popup temporarily?

= Configuration option: =

= How to show popup in specific page/post? =

Use plugin short code to display the popup in specific posts or pages. Just copy and paste the available short code in your post.

= How to show popup in entire website? =

Go to widget page and drag and drop the widget IFrame popup into your sidebar, so that popup will display in entire website. (or) Just copy plugin PHP code in footer.php file.

= How to show popup only on home page? =

Add the plugin PHP code in your theme footer.php file.

== Installation ==

[Installation Instruction and Configuration](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/)

== Frequently Asked Questions ==

*   Q1. How to show popup in specific page/post?
*   Q2. How to show popup in entire website?
*   Q3. How to show popup only on home page?
*   Q4. How to show popup in entire website except home page?
*   Q5. How to display iframe popup window once per session?
*   Q6. What is CATEGORY and ID in the plugin short code?
*   Q7. How to set/modify popup window size?
*   Q8. How to hide/disable popup temporarily?

FAQ Answer [http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/](http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/)

== Screenshots ==

1. Front Page. http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/

2. Admin Page. http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/

== Changelog ==

= 1.0 =

*  First version

= 1.1 =

*  As per wp plugin team advice default link changed to http://www.example.com/

= 1.2 =

*  Tested up to 3.9
*  Listed plugin on WordPress directory.

= 1.3 =

* jscolor picker added for Overlay Color textbox.

= 1.4 =

1. Fixed PHP warning message in JavaScript registration method.
2. In Category dropdown box, the category list has been increased from 10 to 15.

= 1.5 =

*  Tested up to 4.0

= 1.6 =

*  Tested up to 4.1

= 1.7 =

1. Tested up to 4.2.2

= 1.8 =

1. Tested up to 4.3

== Upgrade Notice ==

= 1.0 =

*  First version

= 1.1 =

*  As per wp plugin team advice default link changed to http://www.example.com/

= 1.2 =

*  Tested up to 3.9
*  Listed plugin on WordPress directory.

= 1.3 =

* jscolor picker added for Overlay Color textbox.

= 1.4 =

1. Fixed PHP warning message in JavaScript registration method.
2. In Category dropdown box, the category list has been increased from 10 to 15.

= 1.5 =

*  Tested up to 4.0

= 1.6 =

*  Tested up to 4.1

= 1.7 =

1. Tested up to 4.2.2

= 1.8 =

1. Tested up to 4.3

changelog.txt
Plugin Name	: iframe popup
Plugin URI	: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Author		: Gopi Ramasamy
Author URI	: http://www.gopiplus.com
License		: GPLv2 or later
License URI	: http://www.gnu.org/licenses/gpl-2.0.html
 
********************************Version 1.0************************************************************

First version.

1. Lightweight JQuery (fancybox extension).
2. Timeout to open popup.
3. Option to configure once per session.
4. Responsive.

********************************Version 1.1************************************************************

Default link changed to http://www.example.com/

File changed
File Name : classes/iframe-query.php
Method : public static function popup_default()

********************************Version 1.2************************************************************

1. Tested up to 3.9
2. Listed plugin on WordPress directory.

********************************Version 1.3************************************************************

1. jscolor picker added for Overlay Color textbox.

********************************Version 1.4************************************************************

1. Fixed PHP warning message in JavaScript registration method.
2. In Category dropdown box, the category list has been increased from 10 to 15.

********************************Version 1.5************************************************************

1. Tested up to 4.0

********************************Version 1.6************************************************************

1. Tested up to 4.1

********************************Version 1.7************************************************************

1. Tested up to 4.2.2

********************************Version 1.8************************************************************

1. Tested up to 4.3

iframe-popup.php
<?php
/*
Plugin Name: iframe popup
Plugin URI: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Description: Iframe popup plugin is specially developed to display any webpage in the popup window using web URL. Iframe popup uses JQuery fancybox extension to display popup in iframe window. This plugin will help you to display popup window easily in your blog. You can easily customize the fancybox popup attributes in the plugin admin page.
Version: 1.8
Author: Gopi Ramasamy
Donate link: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
Author URI: http://www.gopiplus.com/work/2014/04/13/iframe-popup-wordpress-plugin/
License: GPLv2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html
*/

/*  
Copyright 2015 iframe popup (www.gopiplus.com)

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License, version 2, as 
published by the Free Software Foundation.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
*/

if(preg_match('#' . basename(__FILE__) . '#', $_SERVER['PHP_SELF'])) { die('You are not allowed to call this page directly.'); }

if (!session_id()) { session_start(); }

require_once(dirname(__FILE__).DIRECTORY_SEPARATOR.'classes'.DIRECTORY_SEPARATOR.'iframe-stater.php');
add_action('admin_menu', array( 'iframepopup_cls_registerhook', 'iframepopup_adminmenu' ));
register_activation_hook(IFRAMEPOP_FILE, array( 'iframepopup_cls_registerhook', 'iframepopup_activation' ));
register_deactivation_hook(IFRAMEPOP_FILE, array( 'iframepopup_cls_registerhook', 'iframepopup_deactivation' ));
add_action( 'widgets_init', array( 'iframepopup_cls_registerhook', 'iframepopup_widget_loading' ));
add_shortcode( 'iframe-popup', 'iframepopup_shortcode' );
add_action('wp_enqueue_scripts', 'iframepopup_add_javascript_files');

function iframepopup_textdomain() 
{
	  load_plugin_textdomain( 'iframe-popup' , false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
}
add_action('plugins_loaded', 'iframepopup_textdomain');
?>

page/popup-show.php
<?php if(preg_match('#' . basename(__FILE__) . '#', $_SERVER['PHP_SELF'])) { die('You are not allowed to call this page directly.'); } ?>
<?php
// Form submitted, check the data
if (isset($_POST['frm_iframepopup_display']) && $_POST['frm_iframepopup_display'] == 'yes')
{
	$did = isset($_GET['did']) ? $_GET['did'] : '0';
	if(!is_numeric($did)) { die('<p>Are you sure you want to do this?</p>'); }
	
	$iframepopup_success = '';
	$iframepopup_success_msg = FALSE;
	
	// First check if ID exist with requested ID
	$result = iframepopup_cls_dbquery::popup_count($did);
	
	if ($result != '1')
	{
		?><div class="error fade"><p><strong><?php _e('Oops, selected details doesnt exist.', IFRAMEPOP_TDOMAIN); ?></strong></p></div><?php
	}
	else
	{
		// Form submitted, check the action
		if (isset($_GET['ac']) && $_GET['ac'] == 'del' && isset($_GET['did']) && $_GET['did'] != '')
		{
			//	Just security thingy that wordpress offers us
			check_admin_referer('iframepopup_form_show');
			
			//	Delete selected record from the table
			iframepopup_cls_dbquery::popup_delete($did);
			
			//	Set success message
			$iframepopup_success_msg = TRUE;
			$iframepopup_success = __('Selected record was successfully deleted.', IFRAMEPOP_TDOMAIN);
		}
	}
	
	if ($iframepopup_success_msg == TRUE)
	{
		?><div class="updated fade"><p><strong><?php echo $iframepopup_success; ?></strong></p></div><?php
	}
}
?>
<div class="wrap">
  <div id="icon-edit" class="icon32 icon32-posts-post"></div>
    <h2><?php _e(IFRAMEPOP_PLUGIN_DISPLAY, IFRAMEPOP_TDOMAIN); ?>
	<a class="add-new-h2" href="<?php echo IFRAMEPOP_ADMINURL; ?>&page=iframe-popup&ac=add"><?php _e('Add New', IFRAMEPOP_TDOMAIN); ?></a></h2>
    <div class="tool-box">
	<?php
		$myData = array();
		$myData = iframepopup_cls_dbquery::popup_select(0);
		?>
		<script language="JavaScript" src="<?php echo IFRAMEPOP_URL; ?>page/setting.js"></script>
		<form name="frm_iframepopup_display" method="post">
      <table width="100%" class="widefat" id="straymanage">
        <thead>
          <tr>
			<th scope="col"><?php _e('Id', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Title', IFRAMEPOP_TDOMAIN); ?></th>
            <th scope="col"><?php _e('URL', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Width', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Height', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Start Date', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Expiration', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Category', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Overlay Color', IFRAMEPOP_TDOMAIN); ?></th>
          </tr>
        </thead>
		<tfoot>
          <tr>
			<th scope="col"><?php _e('Id', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Title', IFRAMEPOP_TDOMAIN); ?></th>
            <th scope="col"><?php _e('URL', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Width', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Height', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Start Date', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Expiration', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Category', IFRAMEPOP_TDOMAIN); ?></th>
			<th scope="col"><?php _e('Overlay Color', IFRAMEPOP_TDOMAIN); ?></th>
          </tr>
        </tfoot>
		<tbody>
			<?php 
			$i = 0;
			if(count($myData) > 0 )
			{
				foreach ($myData as $data)
				{
					?>
					<tr class="<?php if ($i&1) { echo'alternate'; } else { echo ''; }?>">
						<td><?php echo $data['id']; ?></td>
						<td><?php echo stripslashes(substr($data['title'], 0, 25)); ?>...
						<div class="row-actions">
						<span class="edit">
						<a title="Edit" href="<?php echo IFRAMEPOP_ADMINURL; ?>&ac=edit&amp;did=<?php echo $data['id']; ?>"><?php _e('Edit', IFRAMEPOP_TDOMAIN); ?></a> | </span>
						<span class="trash">
						<a onClick="javascript:_iframepopup_delete('<?php echo $data['id']; ?>')" href="javascript:void(0);"><?php _e('Delete', IFRAMEPOP_TDOMAIN); ?></a></span> 
						</div>
						</td>
						<td><a target="_blank" href="<?php echo $data['url']; ?>"><img src="<?php echo IFRAMEPOP_URL; ?>image/link_icon.gif" /></a></td>
						<td><?php echo $data['width']; ?></td>
						<td><?php echo $data['height']; ?></td>
						<td><?php echo substr($data['starttime'],0,10); ?></td>
						<td><?php echo substr($data['expiration'],0,10); ?></td>
						<td><?php echo $data['group']; ?></td>
						<td><?php echo $data['overlaycolor']; ?></td>
					</tr>
					<?php 
					$i = $i+1; 
				} 	
			}
			else
			{
				?><tr><td colspan="9" align="center"><?php _e('No records available.', IFRAMEPOP_TDOMAIN); ?></td></tr><?php 
			}
			?>
		</tbody>
        </table>
		<?php wp_nonce_field('iframepopup_form_show'); ?>
		<input type="hidden" name="frm_iframepopup_display" value="yes"/>
      </form>	
	  <div class="tablenav">
	  <h2>
	  <a class="button add-new-h2" href="<?php echo IFRAMEPOP_ADMINURL; ?>&amp;ac=add"><?php _e('Add New', IFRAMEPOP_TDOMAIN); ?></a>
	  <a class="button add-new-h2" href="<?php echo IFRAMEPOP_ADMINURL; ?>&amp;ac=set"><?php _e('Session Setting', IFRAMEPOP_TDOMAIN); ?></a>
	  <a class="button add-new-h2" target="_blank" href="<?php echo IFRAMEPOP_FAV; ?>"><?php _e('Help', IFRAMEPOP_TDOMAIN); ?></a>
	  </h2>
	  </div>
	  <div style="height:5px"></div>
	<h3><?php _e('Plugin configuration option', IFRAMEPOP_TDOMAIN); ?></h3>
	<ol>
		<li><?php _e('Drag and drop the widget (Display entire website)', IFRAMEPOP_TDOMAIN); ?>.</li>
		<li><?php _e('Add popup into specific  post or page using short code', IFRAMEPOP_TDOMAIN); ?></li>
		<li><?php _e('Add directly in to the theme using PHP code', IFRAMEPOP_TDOMAIN); ?></li>
	</ol>
	<p class="description"><?php echo IFRAMEPOP_OFFICIAL; ?></p>
	</div>
</div>

page/popup-setting.php
<?php if(preg_match('#' . basename(__FILE__) . '#', $_SERVER['PHP_SELF'])) { die('You are not allowed to call this page directly.'); } ?>
<div class="wrap">
  <div class="form-wrap">
    <div id="icon-edit" class="icon32 icon32-posts-post"></div>
    <h2><?php _e(IFRAMEPOP_PLUGIN_DISPLAY, IFRAMEPOP_TDOMAIN); ?></h2>
    <?php
	$iframepopup_session = get_option('iframepopup_session');

	if (isset($_POST['iframepopup_form_submit']) && $_POST['iframepopup_form_submit'] == 'yes')
	{
		//	Just security thingy that wordpress offers us
		check_admin_referer('iframepopup_form_setting');
			
		$iframepopup_session = stripslashes($_POST['iframepopup_session']);	
		update_option('iframepopup_session', $iframepopup_session );
		?>
		<div class="updated fade">
			<p><strong><?php _e('Details was successfully updated.', IFRAMEPOP_TDOMAIN); ?></strong></p>
		</div>
		<?php
	}
	?>
	<script language="JavaScript" src="<?php echo IFRAMEPOP_URL; ?>page/setting.js"></script>
	<h3><?php _e('Session Setting', IFRAMEPOP_TDOMAIN); ?></h3>
	<form name="iframepopup_form_setting" method="post" action="#" onsubmit="return _iframepopup_submit_setting()">
		<label for="tag-title"><?php _e('Session option (Global setting)', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="iframepopup_session" id="iframepopup_session">
            <option value=''>Select</option>
			<option value='NO' <?php if($iframepopup_session == 'NO') { echo 'selected' ; } ?>>NO</option>
            <option value='YES' <?php if($iframepopup_session == 'YES') { echo 'selected' ; } ?>>YES</option>
          </select>
		<p><?php _e('Select YES to show popup once per session, Meaning, popup never appear again if user navigate to another page.', IFRAMEPOP_TDOMAIN); ?></p>
				
		<div style="height:10px;"></div>
		<input type="hidden" name="iframepopup_form_submit" value="yes"/>
		<input name="iframepopup_submit" id="iframepopup_submit" class="button" value="<?php _e('Submit', IFRAMEPOP_TDOMAIN); ?>" type="submit" />
		<input name="publish" lang="publish" class="button" onclick="_iframepopup_redirect()" value="<?php _e('Cancel', IFRAMEPOP_TDOMAIN); ?>" type="button" />
		<input name="Help" lang="publish" class="button" onclick="_iframepopup_help()" value="<?php _e('Help', IFRAMEPOP_TDOMAIN); ?>" type="button" />
		<?php wp_nonce_field('iframepopup_form_setting'); ?>
	</form>
  </div>
  <br />
  <p class="description"><?php echo IFRAMEPOP_OFFICIAL; ?></p>
</div>

page/popup-add.php
<?php if(preg_match('#' . basename(__FILE__) . '#', $_SERVER['PHP_SELF'])) { die('You are not allowed to call this page directly.'); } ?>
<div class="wrap">
<?php
$iframepopup_errors = array();
$iframepopup_success = '';
$iframepopup_error_found = FALSE;

// Preset the form fields
$form = array(
	'id' => '',
	'title' => '',
	'url' => '',
	'width' => '',
	'height' => '',
	'transitionin' => '',
	'transitionout' => '',
	'centeronscroll' => '',
	'titleshow' => '',
	'expiration' => '',
	'starttime' => '',
	'overlaycolor' => '',
	'group' => '',
	'timeout' => ''
);

// Form submitted, check the data
if (isset($_POST['iframepopup_form_submit']) && $_POST['iframepopup_form_submit'] == 'yes')
{
	//	Just security thingy that wordpress offers us
	check_admin_referer('iframepopup_form_add');
	
	$form['title'] = isset($_POST['title']) ? $_POST['title'] : '';
	if ($form['title'] == '')
	{
		$iframepopup_errors[] = __('Enter your title for popup window.', IFRAMEPOP_TDOMAIN);
		$iframepopup_error_found = TRUE;
	}
	
	$form['url'] = isset($_POST['url']) ? $_POST['url'] : '';
	if ($form['url'] == '')
	{
		$iframepopup_errors[] = __('Enter URL to display in Iframe popup window. URL must start with either http or https', IFRAMEPOP_TDOMAIN);
		$iframepopup_error_found = TRUE;
	}

	$form['width'] = isset($_POST['width']) ? $_POST['width'] : '';
	$form['height'] = isset($_POST['height']) ? $_POST['height'] : '';
	$form['transitionin'] = isset($_POST['transitionin']) ? $_POST['transitionin'] : '';
	$form['transitionout'] = isset($_POST['transitionout']) ? $_POST['transitionout'] : '';
	$form['centeronscroll'] = isset($_POST['centeronscroll']) ? $_POST['centeronscroll'] : '';
	$form['titleshow'] = isset($_POST['titleshow']) ? $_POST['titleshow'] : '';
	$form['expiration'] = isset($_POST['expiration']) ? $_POST['expiration'] : '';
	$form['starttime'] = isset($_POST['starttime']) ? $_POST['starttime'] : '';
	$form['overlaycolor'] = isset($_POST['overlaycolor']) ? $_POST['overlaycolor'] : '';
	$form['group'] = isset($_POST['group']) ? $_POST['group'] : '';
	$form['timeout'] = isset($_POST['timeout']) ? $_POST['timeout'] : '';

	//	No errors found, we can add this Group to the table
	if ($iframepopup_error_found == FALSE)
	{
		$action = iframepopup_cls_dbquery::popup_act($form, "ins");
		if($action == "sus")
		{
			$iframepopup_success = __('New details was successfully added.', IFRAMEPOP_TDOMAIN);
		}
		elseif($action == "err")
		{
			$iframepopup_success = __('Oops unexpected error occurred.', IFRAMEPOP_TDOMAIN);
			$iframepopup_error_found = TRUE;
		}

		// Reset the form fields
		$form = array(
			'id' => '',
			'title' => '',
			'url' => '',
			'width' => '',
			'height' => '',
			'transitionin' => '',
			'transitionout' => '',
			'centeronscroll' => '',
			'titleshow' => '',
			'expiration' => '',
			'starttime' => '',
			'overlaycolor' => '',
			'group' => '',
			'timeout' => ''
		);
	}
}

if ($iframepopup_error_found == TRUE && isset($iframepopup_errors[0]) == TRUE)
{
	?>
	<div class="error fade">
		<p><strong><?php echo $iframepopup_errors[0]; ?></strong></p>
	</div>
	<?php
}
if ($iframepopup_error_found == FALSE && strlen($iframepopup_success) > 0)
{
	?>
	  <div class="updated fade">
		<p><strong><?php echo $iframepopup_success; ?> <a href="<?php echo IFRAMEPOP_ADMINURL; ?>"><?php _e('Click here', IFRAMEPOP_TDOMAIN); ?></a> 
			<?php _e('to view the details', IFRAMEPOP_TDOMAIN); ?></strong></p>
	  </div>
	  <?php
	}
?>
<script language="JavaScript" src="<?php echo IFRAMEPOP_URL; ?>page/setting.js"></script>
<script language="JavaScript" src="<?php echo IFRAMEPOP_URL; ?>inc/color/jscolor.js"></script>
<div class="form-wrap">
	<div id="icon-edit" class="icon32 icon32-posts-post"><br></div>
	<h2><?php _e(IFRAMEPOP_PLUGIN_DISPLAY, IFRAMEPOP_TDOMAIN); ?></h2>
	<form name="iframepopup_form" method="post" action="#" onsubmit="return _iframepopup_submit()"  >
      <h3><?php _e('Add details', IFRAMEPOP_TDOMAIN); ?></h3>
      
	  	<label for="tag-a"><?php _e('Iframe URL', IFRAMEPOP_TDOMAIN); ?></label>
		<input name="url" type="text" id="url" value="" size="70" maxlength="255" />
		<p><?php _e('Enter URL to display in Iframe popup window. URL must start with either http or https.', IFRAMEPOP_TDOMAIN); ?><br />Example: http://www.gopiplus.com/</p>
		
		<label for="tag-a"><?php _e('Popup title', IFRAMEPOP_TDOMAIN); ?></label>
		<input name="title" type="text" id="title" value="" size="70" maxlength="255" />
		<p><?php _e('Enter your title for popup window.', IFRAMEPOP_TDOMAIN); ?></p>

		<label for="tag-a"><?php _e('Width', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="width" id="width">
			<option value='30%'>30%</option>
			<option value='35%'>35%</option>
			<option value='40%'>40%</option>
			<option value='45%'>45%</option>
			<option value='50%'>50%</option>
			<option value='55%'>55%</option>
			<option value='60%' selected="selected">60%</option>
			<option value='65%'>65%</option>
			<option value='70%'>70%</option>
			<option value='75%'>75%</option>
			<option value='80%'>80%</option>
			<option value='85%'>85%</option>
			<option value='90%'>90%</option>
		</select>
		<p><?php _e('Select your width percentage for popup window.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Height', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="height" id="height">
			<option value='30%'>30%</option>
			<option value='35%'>35%</option>
			<option value='40%'>40%</option>
			<option value='45%'>45%</option>
			<option value='50%'>50%</option>
			<option value='55%'>55%</option>
			<option value='60%' selected="selected">60%</option>
			<option value='65%'>65%</option>
			<option value='70%'>70%</option>
			<option value='75%'>75%</option>
			<option value='80%'>80%</option>
			<option value='85%'>85%</option>
			<option value='90%'>90%</option>
		</select>
		<p><?php _e('Select your height percentage for popup window.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Transition In', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="transitionin" id="transitionin">
			<option value='fade' selected="selected">fade</option>
			<option value='elastic'>elastic</option>
			<option value='none'>none</option>
		</select>
		<p><?php _e('Transition type while opening popup window.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Transition Out', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="transitionout" id="transitionout">
			<option value='fade' selected="selected">fade</option>
			<option value='elastic'>elastic</option>
			<option value='none'>none</option>
		</select>
		<p><?php _e('Transition type while closing popup window.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Center on scroll', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="centeronscroll" id="centeronscroll">
			<option value='true' selected="selected">true</option>
			<option value='false'>false</option>
		</select>
		<p><?php _e('If true, popup window is centered while scrolling page.', IFRAMEPOP_TDOMAIN); ?></p>

		<label for="tag-a"><?php _e('Show Title', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="titleshow" id="titleshow">
			<option value='true' selected="selected">YES</option>
			<option value='false'>NO</option>
		</select>
		<p><?php _e('Display title under popup window.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Start Date', IFRAMEPOP_TDOMAIN); ?></label>
		<input name="starttime" type="text" id="starttime" value="2014-04-01" maxlength="10" />
		<p><?php _e('Please enter popup display start date in this format YYYY-MM-DD', IFRAMEPOP_TDOMAIN); ?></p>			
		
		<label for="tag-a"><?php _e('Expiration Date', IFRAMEPOP_TDOMAIN); ?></label>
		<input name="expiration" type="text" id="expiration" value="9999-12-31" maxlength="10" />
		<p><?php _e('Please enter popup expiration date in this format YYYY-MM-DD', IFRAMEPOP_TDOMAIN); ?></p>	
		
		<label for="tag-a"><?php _e('Overlay Color', IFRAMEPOP_TDOMAIN); ?></label>
		<input class="color" type="text" name="overlaycolor" id="overlaycolor" value="#666666" maxlength="7" />
		<p><?php _e('Color of the overlay for popup window. (Example: #666666)', IFRAMEPOP_TDOMAIN); ?></p>	
		
		<label for="tag-a"><?php _e('Category', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="group" id="group">
			<?php for($i=1; $i<=15; $i++) { ?>
				<option value='Category<?php echo $i; ?>'>Category<?php echo $i; ?></option>
			<?php } ?>
		</select>
		<p><?php _e('Select category for this popup content.', IFRAMEPOP_TDOMAIN); ?></p>
		
		<label for="tag-a"><?php _e('Timeout', IFRAMEPOP_TDOMAIN); ?></label>
		<select name="timeout" id="timeout">
			<option value='1000'>1 Sec</option>
			<option value='2000'>2 Sec</option>
			<option value='4000' selected="selected">4 Sec</option>
			<option value='6000'>6 Sec</option>
			<option value='8000'>8 Sec</option>
			<option value='10000'>10 Sec</option>
			<option value='12000'>12 Sec</option>
			<option value='15000'>15 Sec</option>
			<option value='20000'>20 Sec</option>
		</select>
		<p><?php _e('Timeout to open popup window.', IFRAMEPOP_TDOMAIN); ?></p>
	  
      <input name="id" id="id" type="hidden" value="">
      <input type="hidden" name="iframepopup_form_submit" value="yes"/>
      <p class="submit">
        <input name="publish" lang="publish" class="button" value="<?php _e('Submit', IFRAMEPOP_TDOMAIN); ?>" type="submit" />
        <input name="publish" lang="publish" class="button" onclick="_iframepopup_redirect()" value="<?php _e('Cancel', IFRAMEPOP_TDOMAIN); ?>" type="button" />
        <input name="Help" lang="publish" class="button" onclick="_iframepopup_help()" value="<?php _e('Help', IFRAMEPOP_TDOMAIN); ?>" type="button" />
      </p>
	  <?php wp_nonce_field('iframepopup_form_add'); ?>
    </form>
</div>
<p class="description"><?php echo IFRAMEPOP_OFFICIAL; ?></p>
</div>